LICENSE

```diff
@@ -200,3 +200,14 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+--------------------------------------------------------------------------------
+
+/src/main/java/com/google/cloud/spark/bigquery/converters/ArrowSchemaConverter.java
```

Comment:
@davidrabinowitz @emkornfield Could you please review this retribution?

Reply:
Looks reasonable to me.

Reply:
I've got the response from Google legal - in your e-mail.

Reply:
Updated folder structure